packages/ui/src/theme/css/component/sliderField.scss

```diff
@@ -15,36 +15,42 @@
   touch-action: none;
   box-sizing: content-box; // we want the padding to add to the height
   padding-block: var(--amplify-components-sliderfield-padding-block);
+  --amplify-internal-sliderfield__root-height: var(
```

Comment:
What does it mean that these are `--amplify-internal`?

Reply:
these new `--internal` css variables are being used to keep the css structure flat.  Some of our components change different values based on the different states they are in for example the `large` modifier on a divider will change a different value when the divider is horizontal vs vertical.  These new `--internal` variables are used to update these values for the different states they are in.